en-US/Players_Guide/03_Creating_a_Character/Classes/Adept/Adept.md

```diff
@@ -1,4 +1,234 @@
+
+Note: This wording might not suffice to define the category, but prefer positive wording over negative where it can be used.  And kick the restrictions part to the end of the explanation, rather than letting it get in the way of understanding the core benefit -- especially since most players will easily grasp "monk doesn't wear armor or she loses her abilities" and not need to have it pointed out a lot.  In fact it might fit a sidebar just by itself, pointing out which abilities get nerfed when you're wearing armor.
+
+"Prefer positive over negative" is a bit at odds with me rephrasing other things from positive **to** negative.
+
+I think what I'm trying for is to state the base case, and **then** the exceptions.  "You do X, but lose this ability if you J or K or L" reads more cleanly than "If you !J and !K and !L then you can X."
+-->
+
+You gain the following benefits while you are unarmed or wielding only Adept weapons and you aren’t wearing armor or wielding a shield:
+
+- You can use Dexterity instead of Strength for the attack and damage rolls of your unarmed strikes and Adept weapons.
+- You can roll a d4 in place of the normal damage of your unarmed strike or Adept weapon.
+  This die changes as you gain Adept levels, as shown in the Martial Arts column of the [Adept table](#Adept_the_adept_table).
+- When you use the Attack action with an unarmed strike or an Adept weapon on your turn, you can make one unarmed strike as a bonus action.
+  For example, if you take the Attack action and attack with a quarterstaff, you can also make an unarmed strike as a bonus action, assuming you haven’t already taken a bonus action this turn.
+
+Certain monasteries use specialized forms of the Adept weapons. For example, you might use a club that is two lengths of wood connected by a short chain (called a nunchaku) or a sickle with a shorter, straighter blade (called a kama).
+Whatever name you use for an Adept weapon, you can use the game statistics provided for the weapon.
+
+###### Focus (2nd Level) {#Adept_focus}
+
+Starting at 2nd level, your training allows you to harness the mystic energy of focus points.
+Your access to this energy is represented by a number of focus points.
+Your Adept level determines the number of points you have, as shown in the Focus Points column of the [Adept table](#Adept_the_adept_table).
+
+You can spend these points to fuel various focus features. You start knowing three such features: _Flurry of Blows_, Patient Defense_, and _Step of the Wind_.
+
+- **Flurry of Blows (1 focus point).**
+  Immediately after you take the Attack action on your turn, make two unarmed strikes as a bonus action.
+- **Patient Defense (1 focus point).**
+  Take the Dodge action as a bonus action on your turn.
+- **Step of the Wind (1 focus point).**
+  Take the Disengage or Dash action as a bonus action on your turn, and your jump distance is doubled for the turn.
```

Comment:
I moved these options up, so that they are closer to the text that explains what they're about.
I also modified the structure to be closer to that used in the "Disciple of the Outcast".